templates/frontend/pages/indexJournal.tpl

```diff
@@ -21,9 +21,21 @@
 {include file="frontend/components/header.tpl" pageTitleTranslated=$currentJournal->getLocalizedName()}
 
 <main class="page_index_journal">
-	{* Display homepage image if set *}
+
+	{* Display homepage image if set, and wrap around journal summary if use chooses to display it *}
+	{if $homepageImage}
+		<div class="homepage_image" style="background-image: url('{$publicFilesDir}/{$homepageImage.uploadName|escape:"url"}')">
+	{/if}
+
+	{if $showJournalSummary}
+		<section class="container journal_summary">
+			<h2>{translate key="navigation.about"}</h2>
+			{$currentJournal->getLocalizedDescription()}
```

Comment:
I see from the page description that we already have variable `$journalDescription` assigned to this template: https://github.com/pkp/classic/blob/49cc3cedc7fe590b3a4b4a42dbb10370db2c154a/templates/frontend/pages/indexJournal.tpl#L11
Just want to add that usually, we escape input that comes from TinyMCE forms with `|strip_unsafe_html`. But in this case, as it comes from the settings form, we can trust the input - it's accessible only for administators and journal managers. Thus, you are right, no need to escape. Can you also look if we should use `nl2br` modifier here?

Reply:
Is it possible that this page’s description was copy-pasted from a different theme? the `$journalDescription` variable doesn’t work and isn’t found anywhere in `ClassicThemePlugin.inc.php` either.

Reply:
Also, shouldn’t this be a specific variable for cases when users want to show the journal description, not when a journal description has been entered?

Reply:
When a template says `@uses $journalDescription` it means the variable is fed to the template by the backend. So even if it's not used it should be available.

However, my preference is to abandon these configured variables and use `{$currentJournal}` whenever possible, as you've done here.

> Also, shouldn’t this be a specific variable for cases when users want to show the journal description, not just when a journal description has been entered?

I think the trick is to use both in the conditional. So line 30 would look like this:

```html
{if $showJournalSummary && $currentJournal->getLocalizedDescription()}
```

That way if they have not filled out the description we won't see "About" without any additional text.

Reply:
Oh, i see that Vitaliy has already suggested the conditional change in https://github.com/pkp/classic/pull/37/files/c3cda705704e20b185cee6ec9bde071e1b9cc9bf..49cc3cedc7fe590b3a4b4a42dbb10370db2c154a#diff-e89fda81f995b935ee0e12b739c3e119

Reply:
@NateWr I did this in the following commit -- https://github.com/pkp/classic/pull/37/commits/6a1b3842301f06db15c94d3495916d786067a59e 

Hopefully that’s what was missing. I have also pushed the same commit in the Immersion theme, here: https://github.com/pkp/immersion/pull/20/commits/254f621168a83571a2e7a4a77bd7c2f658baf426